docs/tutorials/integrations/ghactions.md

```diff
@@ -4,10 +4,10 @@ template: main.html
 
 # Use Iter8 in a GitHub Actions workflow
 
-Install the latest version of Iter8 using the GitHub Action `iter8-tools/iter8@v0.10`. Once installed, Iter8 can be used as documented (see [user guide](../../user-guide/commands/iter8.md)) in `run` actions. For example:
+Install the latest version of Iter8 CLI using the GitHub Action `iter8-tools/iter8@v0.11`. Once installed, Iter8 CLI can be used as documented in the [tutorials](../../getting-started/your-first-experiment.md) and [user guide](../../user-guide/commands/iter8.md). For example:
 
 ```yaml
-- uses: iter8-tools/iter8@v0.10 # install Iter8
+- uses: iter8-tools/iter8@v0.11 # install Iter8
 - run: |
     iter8 version
     iter8 launch -c load-test-http \
```

Comment:
Suggest changing this to match the new approach:  `--set "tasks={http,assess}"` instead of `-c`.